src/libraries/Microsoft.PowerFx.Core/Binding/BindingConfig.cs

```diff
@@ -17,12 +17,15 @@ internal class BindingConfig
 
         public bool AllowsSideEffects { get; }
 
-        public bool UseThisRecordForRuleScope { get; }
+        public bool UseThisRecordForRuleScope { get; }
+
+        public bool NumberIsFloat { get; }
 
```

Comment:
Should this be on binding config, or should it be on Features?

Reply:
Binding Config is currently just used for permanent flags, not 1.0 features.

Reply:
I see NumberIsFloat as a permanent feature.  Folks like Azure IOT will never want the performance hit of working with decimal.  For this reason, I moved it out of features and into the config here, in the parser, and in the lexer.  Totally open to having this discussion.